src/components/Products.js

```diff
@@ -1,39 +1,43 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import ProductCardSmall from './ProductCardSmall'
-import { fetchAllBeverages } from '../actions/beverageActions';
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import ProductCardSmall from "./ProductCardSmall";
+import { fetchAllBeverages } from "../actions/beverageActions";
 
 class Products extends Component {
+  componentDidMount() {
```

Comment:
Please remove the fetch beverages logic from Products.js. You can do this manually or merge in dev to your local branch.

Reply:
Okay it has been removed